akka-stream-typed/src/test/scala/docs/scaladsl/ActorFlowSpec.scala

```diff
@@ -42,6 +45,24 @@ class ActorFlowSpec extends ScalaTestWithActorTestKit with AnyWordSpecLike {
         Behaviors.same
     })
 
+    val replierWithSuccess = spawn(Behaviors.receiveMessage[AskingWithStatus] {
+      case AskingWithStatus("TERMINATE", _) =>
+        Behaviors.stopped
```

Comment:
The new tests are not using the "TERMINATE" message, so we can remove this. 

It's used by the `replier` one to verify the stream stops. We don't need to test it twice since `ActorFlow.askWithStatus` is built on top of `ActorFlow.ask`.